### DISCUSSION TITLE
Project unusable on mobile

### DISCUSSION THREAD

**Discussion by Slion** [community/community#69167] [Projects and Issues]
### Select Topic Area

Bug

### Body

On a Project page on mobile you can't scroll through the issues without dragging them. Pretty sure this is a regression as it used to work fine. I just can't tell you when it happened cause it's been months since I used Project on mobile. As you scroll through your todo the shadow of the item you started to scroll from remains on the screen while you scroll. That makes it hard to read. Then you are very likely to drag that shadow as you keep scrolling and drop it somewhere you did not intend to.

This ought to be fixed ASAP as it's not only unusable, it is also causing users to mess up the project priorities, meaning user data loss.

I can reproduce that on a WebView browser like Fulguris and on Chrome.

**Comment by anoospaapay** [comment]
ً

**Comment by taki-on** [comment]
@Slion Thank you for reporting the issue. I wonder if you have experienced the issue on the mobile web. Our native mobile apps do not support drag and drop of project items. If the issue is about the mobile web, I will transfer the issue to the corresponding team.

**Reply by Slion** [reply]
Mobile web

**Reply by jsoref** [reply]
@taki-on you probably meant to update the category.

**Reply by Hinny** [reply]
Any progress on this issue? It has been over a year. If you don't know what the issue is, just create a small project with 10 issues in a couple of columns and try to navigate around.

**Comment by halfpasttense** [comment]
And 10 months later this is still an issue that just caused me to mess up my project while simply trying to add a draft task. I wouldn't have even had to try this on the mobile web version if the feature was actually available on the mobile app. Mind boggling that it's not.

**Comment by Hinny** [comment]
December 2024 and it is still an issue. Please prioritize this, as it super frusrating to use projects in web interface. Just make a drag-and-drop handle or button to start rearranging, or anything but this. It is worse than unusable in its current state, since it messes up the boards.
